tests/output/ConsoleInteractionTest.py

```diff
@@ -439,8 +473,10 @@ def test_print_result_no_input(self):
                                  """
 Project wide:
 |    | [NORMAL] origin:
-|    | message
-""")
+|    | {}
+""".rstrip("\r\n").format(highlight("message", TextLexer(),
```

Comment:
Oh stupid me. Thanks, I ignored that and was thinking you're talking about the other rstrip. Fixed now